doc/src/manual/networking-and-streams.md

```diff
@@ -216,15 +216,21 @@ TCPServer(active)
 julia> listen(IPv6(0),2001) # Listens on port 2001 on all IPv6 interfaces
 TCPServer(active)
 
-julia> listen("testsocket") # Listens on a UNIX domain socket/named pipe
+julia> listen("testsocket") # Listens on a UNIX domain socket
+PipeServer(active)
```

Comment:
Since these are not exported in the repl these show up as `Base.PipeServer, Base.TCPServer` ....   should I also update these to include the `Base` qualification?